client/src/app/common/CustomRules/add-custom-rules.tsx

```diff
@@ -29,11 +33,37 @@ export const AddCustomRules: React.FC<IAddCustomRules> = ({
   customRulesFiles,
   readFileData,
   setReadFileData,
+  taskgroupID,
 }: IAddCustomRules) => {
   const [error, setError] = React.useState("");
   const [currentFiles, setCurrentFiles] = React.useState<File[]>([]);
   const [showStatus, setShowStatus] = React.useState(false);
 
+  const onUploadError = (error: AxiosError) =>
+    console.log("File upload failed: ", error);
+
+  const { mutate: uploadFile } = useUploadFileMutation(() => {}, onUploadError);
+
+  const onCreateRuleFileSuccess = (
+    response: any,
+    formData: FormData,
+    file: IReadFile
+  ) => {
+    //Set file ID for use in form submit
+    // setValue("imageID", response?.id);
```

Comment:
Is this commented-out code important? We probably want to remove it and open an issue if necessary